server/services/case-service.js

```diff
@@ -83,10 +84,63 @@ const createCaseService = (apiUrl) => {
       }
     },
 
+    getOutcomesList: async (courtCode, selectedFilterSorts, subsection) => {
+      const { filters, sorts } = getOutcomeListFilterSorts(selectedFilterSorts)
+
+      const filterMap = {
```

Comment:
Is it better to use value/label eg.  PROBATION_SENTENCE/'Probation sentence' in the dropdown options so that you don't have to do this mapping?